doc/dev/quick_start.rst

```diff
@@ -741,12 +744,3 @@ as well as vectorized using ``jax.vmap``:
 
 >>> jax.vmap(cost_fn)(jnp.array([[0.1, 0.2, 0.3], [0.4, 0.5, 0.6]]))
 Array([1.32269195, 1.53905377], dtype=float64)
-
-In particular, this allows for a reduction in boilerplate when using
-JAX-compatible optimizers such as ``jaxopt``:
-
->>> opt = jaxopt.GradientDescent(cost_fn)
->>> params = jnp.array([0.1, 0.2, 0.3])
->>> (final_params, _) = jax.jit(opt.run)(params)
->>> final_params
-Array([-0.00320799,  0.03475223,  0.29362844], dtype=float64)
```

Comment:
@tzunghanjuang is there an optax version we could replace this with?

Reply:
I have wrapped up a function for that in this test.